__tests__/Layout.test.js

```diff
@@ -18,7 +18,6 @@ const patterns = [
     codeES5: 'Code ES5 - Prototype',
     codeES6: 'Code ES6 - Prototype',
     answered: false,
-    answerId: null,
     correct: null
```

Comment:
I want to discuss the field `correct` as it is now computed as `uuid === variantUuid`